### ISSUE TITLE
get_format_test too restrictive (digits)

### ISSUE THREAD

**Issue by fergusg** [htmllint/htmllint#90]
The "dash" and "underscore" patterns insist that the sub-parts all start with a letter.  This means that "col-1" isn't allowed.  I think it should be (for example, bootstrap uses them a lot)

``` javascript
var formats = {
    lowercase: /^[a-z][a-z\d]*$/,
    underscore: /^[a-z][a-z\d]*(_[a-z\d]+)*$/,
    dash: /^[a-z][a-z\d]*(-[a-z\d]+)*$/,
    camel: /^[a-zA-Z][a-zA-Z\d]*$/
};
```
